modules/rubiconAnalyticsAdapter.js

```diff
@@ -416,10 +416,6 @@ let rubiconAdapter = Object.assign({}, baseAdapter, {
         }
         bid.clientLatencyMillis = Date.now() - cache.auctions[args.auctionId].timestamp;
         bid.bidResponse = parseBidResponse(args, bid.bidResponse);
-        // RP server banner overwrites bidId with bid.seatBidId
-        if (utils.deepAccess(bid, 'bidResponse.seatBidId') && bid.bidder === 'rubicon' && bid.source === 'server' && ['video', 'banner'].some(i => utils.deepAccess(bid, 'bidResponse.mediaType') === i)) {
-          bid.seatBidId = bid.bidResponse.seatBidId;
-        }
```

Comment:
The line `bid.bidResponse = parseBidResponse(args, bid.bidResponse);`

Adds `seatBidId` to the internal `bidResponse` in the analytics adapter. And if that is present, it overwrites the 
 internal `bidRequest.bidId` which is sent to Analytics.

This `bid.seatBidId` is not referenced or used anywhere else in the code.

So does not make sense to add it to this bid object.